manticore/core/smtlib/solver.py

```diff
@@ -592,7 +597,7 @@ def optimize(self, constraints: ConstraintSet, x: BitVec, goal: str, max_iter=10
                 return last_value
             raise SolverError("Optimizing error, unsat or unknown core")
 
-    def get_value(self, constraints, *expressions):
+    def get_value(self, constraints: ConstraintSet, *expressions):
```

Comment:
This one is also missing a return type hint

Reply:
What's the return type?  I was having trouble figuring out what it is.

`mypy` treats unannotated return types as `Any` return types, so the lack of an explicit annotation there _shouldn't_ be a problem.  Though if we have a more accurate type, that can help both readers & tools.

Reply:
Hmmm. I was hoping typeguard would be able to tell us. We can get to it another time, I guess.

Reply:
You can pass any type in `expressions` and it'll get returned from the values if it's not symbolic.